app/services/proxy_status.rb

```diff
@@ -26,6 +26,18 @@ def tftp_server
     end
   end
 
+  def environment_stats
```

Comment:
I'm a bit concerned that these requests might take a long time and should be handled more quickly on the proxy using https://github.com/theforeman/smart-proxy/pull/356 as a start and improving on it. However if testing indicates that this runs in an acceptable time (a few seconds for a largish setup) that PR can be closed.